scripts/common_startup.sh

```diff
@@ -128,7 +130,7 @@ if [ $SET_VENV -eq 1 ]; then
         echo "ERROR: A virtualenv cannot be found. Please create a virtualenv in $GALAXY_VIRTUAL_ENV, or activate one."
         exit 1
     fi
-elif [ $SET_VENV -eq -0 -a $CONDA_ALREADY_INSTALLED -eq 1 ]; then
+elif [ $SET_VENV -eq 0 -a $CONDA_ALREADY_INSTALLED -eq 1 ]; then
```

Comment:
A good practice in shell scripting is also to always double quote ("...") variables in tests:
```
elif [ "$SET_VENV" -eq 0 -a "$CONDA_ALREADY_INSTALLED" -eq 1 ]; then
```